README.md

```diff
@@ -1,16 +1,16 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Portfolio: setup and mobile version skeleton
-> My Second Project of Converting My Portfolio Website in to Mobile Zise.
+> My Third Project Added Work & About Sections to My Portfolio Website in to Mobile Zise.
```

Comment:
- [ ] Would be really nice if you proofread your documentation it will help fix typo errors